arches_component_lab/src/arches_component_lab/widgets/DateWidget/components/DateWidgetEditor.vue

```diff
@@ -53,19 +67,20 @@ function formatDate(date: Date | null): string | null {
         return null;
     }
 
-    return dayjs(date).format(props.widgetData.config.dateFormat);
+    return dayjs(date).format(props.cardXNodeXWidgetData.config.dateFormat);
 }
 </script>
 
 <template>
     <FormField
-        ref="formFieldRef"
+        ref="formField"
         v-slot="$field"
         :name="props.nodeAlias"
-        :initial-value="props.initialValue"
+        :initial-value="props.value"
         :resolver="resolver"
     >
         <DatePicker
+            :id="`${props.graphSlug}-${props.nodeAlias}-input`"
```

Comment:
it seems like the only reason to pass the graph slug around is to make ids for these inputs, except they aren't truly unique.  Couldn't we just create a unique id and then not have to pass that value down through many, many components.  (eg: DefaultCard --> DefaultCardEditor --> DateWidget -->  GenericWidget --> DateWidgetEditor)

![Screenshot 2025-06-24 at 5 19 18 PM](https://github.com/user-attachments/assets/72f9eb70-88b4-4e5e-b4aa-950fdaef314f)

Reply:
no, `graphSlug` is used in `GenericWidget` to `fetchCardXNodeXWidgetData`. It's passed into LOC in case they would need it for an API call or whatnot. YAGNI I know, but I like the pattern of the widget props being available at all levels of the widget.